### ISSUE TITLE
linux nbd-client error

### ISSUE THREAD

**Issue by akauova** [jeffbryner/NBDServer#12]
Hello.
I have a problem with connecting nbd-client on linux. The error is : 
```
It looks like you're trying to connect to an oldstyle server. This is no longer supported since nbd 3.10
```
How can I fix it? Or will there be updates for the new version of the protocol?

**Comment by Beercow** [comment]
What client and what version are you using?

**Comment by stoner455** [comment]
Sorry, a little late to the game.  The new versions of Linux nbd-client now use a different communication protocol than what the nbd-server Windows exe is using.  You have to install older versions of nbd-client in order to get them to work together.  Not sure if nbd-server can be updated to use the same communication protocols as the new versions of client.

**Comment by davidl2** [comment]
Yes, Linux nbd dropped support for the "oldstyle" protocol in 2015,  see  https://github.com/NetworkBlockDevice/nbd/commit/3694019301fcb5ab05074ea573d5f60f280b2e61  .
